desc/basis.py

```diff
@@ -435,9 +438,10 @@ class FourierSeries(_Basis):
         Maximum toroidal resolution.
     NFP : int
         number of field periods
-    sym : {``'cos'``, ``'sin'``, False}
-        * ``'cos'`` for cos(m*t-n*z) symmetry
-        * ``'sin'`` for sin(m*t-n*z) symmetry
+    sym : {``'cos'``, ``'sin'``, ``'n0'``, False}
+        * ``'cos'`` for cos(n*z) symmetry
+        * ``'sin'`` for sin(n*z) symmetry
+        * ``'n0'`` for no n=0 mode
```

Comment:
I don't love this name. I agree that having a basis that lacks a zero mode is useful, but can we be more verbose and say like `"no n=0"` or something?